src/cdk-experimental/menu/menu.ts

```diff
@@ -136,6 +140,18 @@ export class CdkMenu extends CdkMenuGroup implements Menu, AfterContentInit, OnI
     this._keyManager.setLastItemActive();
   }
 
+  /** Get the tabindex to be applied for this menu. */
+  _getTabIndex(): 0 | null {
+    return this._isInline() ? 0 : null;
+  }
+
+  /** Set focus to the first menu item if this is an inline menu. */
+  _focusFirstItem() {
+    if (this._isInline()) {
```

Comment:
Since `focus` doesn't bubble, you could just always call `this._keyManager.setFirstItemActive();` without checking if the menu is line since this menu element will only even be focusable when it's inline